Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-gem 'jekyll-redirect-from'
+gem "jekyll-redirect-from", "~> 0.16.0"
 gem "jekyll"
```

Comment:
Is there a reason there's no version specified here?

Reply:
Mainly just that I was unsure how it would interact with a specified jekyll version in the gemspec.

Reply:
OK, these are all good questions to answer!